src/Raven.Server/Documents/TimeSeries/TimeSeriesStorage.cs

```diff
@@ -939,12 +964,19 @@ private void ReduceCountBeforeAppend()
                 _tss.Stats.UpdateCountOfExistingStats(_context, SliceHolder, _collection, -ReadOnlySegment.NumberOfLiveEntries);
             }
 
-            public long AppendExistingSegment(TimeSeriesValuesSegment newValueSegment)
+            public long AppendExistingSegment(TimeSeriesValuesSegment newValueSegment, bool fromUpdateStatus = false, string changeVectorFromReplication = null)
```

Comment:
what does it mean `fromUpdateStatus` ?

Reply:
name changed to `conflictStatusIsUpdate` .
this parameter indicates whether we need to generate a new change vector or to accept the remote one (in case of an update)